CI/full-build-macos.sh

```diff
@@ -303,37 +303,68 @@ bundle_dylibs() {
-        -x ./OBS.app/Contents/PlugIns/image-source.so \
-        -x ./OBS.app/Contents/PlugIns/linux-jack.so \
-        -x ./OBS.app/Contents/PlugIns/mac-avcapture.so \
-        -x ./OBS.app/Contents/PlugIns/mac-capture.so \
-        -x ./OBS.app/Contents/PlugIns/mac-decklink.so \
-        -x ./OBS.app/Contents/PlugIns/mac-syphon.so \
-        -x ./OBS.app/Contents/PlugIns/mac-vth264.so \
-        -x ./OBS.app/Contents/PlugIns/mac-virtualcam.so \
-        -x ./OBS.app/Contents/PlugIns/obs-browser.so \
-        -x ./OBS.app/Contents/PlugIns/obs-browser-page \
-        -x ./OBS.app/Contents/PlugIns/obs-ffmpeg.so \
-        -x ./OBS.app/Contents/PlugIns/obs-filters.so \
-        -x ./OBS.app/Contents/PlugIns/obs-transitions.so \
-        -x ./OBS.app/Contents/PlugIns/obs-vst.so \
-        -x ./OBS.app/Contents/PlugIns/rtmp-services.so \
-        -x ./OBS.app/Contents/MacOS/obs-ffmpeg-mux \
-        -x ./OBS.app/Contents/MacOS/obslua.so \
-        -x ./OBS.app/Contents/PlugIns/obs-x264.so \
-        -x ./OBS.app/Contents/PlugIns/text-freetype2.so \
-        -x ./OBS.app/Contents/PlugIns/obs-libfdk.so \
-        -x ./OBS.app/Contents/PlugIns/obs-outputs.so
-    step "Move libobs-opengl to final destination"
+    if ! [ "${CEF_BUILD_VERSION:-${CI_CEF_VERSION}}" -le 3770 ]; then
+        ${CI_SCRIPTS}/app/dylibbundler -cd -of -a ./OBS.app -q -f \
+            -s ./OBS.app/Contents/MacOS \
+            -s "${DEPS_BUILD_DIR}/sparkle/Sparkle.framework" \
+            -s ./rundir/${BUILD_CONFIG}/bin/ \
+            -x ./OBS.app/Contents/PlugIns/coreaudio-encoder.so \
+            -x ./OBS.app/Contents/PlugIns/decklink-ouput-ui.so \
+            -x ./OBS.app/Contents/PlugIns/decklink-captions.so \
+            -x ./OBS.app/Contents/PlugIns/frontend-tools.so \
+            -x ./OBS.app/Contents/PlugIns/image-source.so \
+            -x ./OBS.app/Contents/PlugIns/linux-jack.so \
+            -x ./OBS.app/Contents/PlugIns/mac-avcapture.so \
+            -x ./OBS.app/Contents/PlugIns/mac-capture.so \
+            -x ./OBS.app/Contents/PlugIns/mac-decklink.so \
+            -x ./OBS.app/Contents/PlugIns/mac-syphon.so \
+            -x ./OBS.app/Contents/PlugIns/mac-vth264.so \
+            -x ./OBS.app/Contents/PlugIns/mac-virtualcam.so \
+            -x ./OBS.app/Contents/PlugIns/obs-browser.so \
+            -x ./OBS.app/Contents/PlugIns/obs-ffmpeg.so \
+            -x ./OBS.app/Contents/PlugIns/obs-filters.so \
+            -x ./OBS.app/Contents/PlugIns/obs-transitions.so \
+            -x ./OBS.app/Contents/PlugIns/obs-vst.so \
+            -x ./OBS.app/Contents/PlugIns/rtmp-services.so \
+            -x ./OBS.app/Contents/MacOS/obs-ffmpeg-mux \
+            -x ./OBS.app/Contents/MacOS/obslua.so \
+            -x ./OBS.app/Contents/PlugIns/obs-x264.so \
+            -x ./OBS.app/Contents/PlugIns/text-freetype2.so \
+            -x ./OBS.app/Contents/PlugIns/obs-libfdk.so \
+            -x ./OBS.app/Contents/PlugIns/obs-outputs.so
```

Comment:
I dislike that most of this is duplicated for both cases. Maybe set a variable that you can append to? One less place to modify when adding/removing/changing bundled deps.

Reply:
Just pushed a change, using an array to define plugins and just adding `obs-browser-page` for legacy builds.